pymarl/config/algs/qmix_uc_100.yaml

```diff
@@ -15,7 +15,7 @@ agent_output_type: "q"
 learner: "q_learner"
 double_q: True
 mixer: "qmix"
-mixing_embed_dim: 64
+mixing_embed_dim: 128
```

Comment:
In my case, it works rather well with 64. You found better results with 128?

Reply:
In average yes, but it is also to be consistent with qvmix